source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.js

```diff
@@ -72,7 +72,7 @@ const messages = defineMessages({
   walletsDropdownError: {
     id: 'staking.redeemItnRewards.step1.walletsDropdownError',
     defaultMessage:
-      '!!!The selected wallet does not have sufficient ada to cover the necessary transaction fees. Please choose another wallet or add more funds to this one.',
+      '!!!The selected wallet does not have sufficient ADA to cover the necessary transaction fees. Please choose another wallet or add more funds to this one.',
```

Comment:
@DeeJayElly why is "ada" in this context replaced with "ADA"? This is not correct. It should in "ADA" form ONLY when it is following a number - e.g. 100 ADA...

Reply:
@gnpf reported this as an issue here "On the second validation message, "ada" must be spelled in uppercase. (Step 7)", so i assume this shouldn't be the case @nikolaglumac .